### ISSUE TITLE
Toggle switch help and error text isn't accessible

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#970]
**Describe the bug**
Toggle switch help/error text is not linked to the input via aria-described by like the other form input types.

**Expected behavior**
Toggle switch help and error text should be associated with the input.

**Comment by jamesjacobs** [comment]
Non issue - help/error text is within the wrapping label and therefore `aria-describedby` is unneeded.
